nodewatcher/modules/monitor/http/survey/management/commands/export_survey_data.py

```diff
@@ -136,8 +136,8 @@ def all_nodes_survey_graph(at):
 
     exported_graph = {
         'graph': {
-            'vertices': vertices,
-            'edges': edges,
+            'v': vertices,
+            'e': edges,
```

Comment:
Why this? Those files are not size-sensitive. We are compressing keys in the database because that is an optimization. Here we do not need to optimize anything.